### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by engr-lynx** [RafaelGB/obsidian-db-folder#893]
### Contact Details

techie@lynx.rocks

### What happened?

When sorting using a Calendar type column, it does it alphabetically. The expectation is that the sort happens chronologically.

### What platform were you using?

Desktop

### Version of the plugin

3.4.0

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by GuyUnger** [comment]
yep this and also tags. i have tags that start with numbers 1-10, it shows 10 between 1 and 2. best would be if it uses the order of the tags set by the user, like in notion
